interfaces/cython/cantera/units.pyx

```diff
@@ -1,5 +1,6 @@
 # This file is part of Cantera. See License.txt in the top-level directory or
 # at https://cantera.org/license.txt for license and copyright information.
+from __future__ import annotations
```

Comment:
This may have to go into `_cantera.pyx`?

Reply:
It probably needs to above the comment. I thought it would ignore comments, but I guess not.

Reply:
Well, nothing I tried with this worked. In any case, as of Cython 3.0, this import will not be needed for `*.pyx` files, see: https://github.com/cython/cython/issues/2950